service/src/test/java/org/sunbird/learner/actors/skill/UserSkillManagementActorTest.java

```diff
@@ -0,0 +1,355 @@
+package org.sunbird.learner.actors.skill;
+
+import static akka.testkit.JavaTestKit.duration;
+
+import akka.actor.ActorRef;
+import akka.actor.ActorSystem;
+import akka.actor.Props;
+import akka.testkit.javadsl.TestKit;
+import java.sql.Timestamp;
+import java.util.*;
+import org.junit.*;
+import org.sunbird.cassandra.CassandraOperation;
+import org.sunbird.common.ElasticSearchUtil;
+import org.sunbird.common.exception.ProjectCommonException;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.ActorOperations;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.common.models.util.ProjectUtil;
+import org.sunbird.common.models.util.datasecurity.OneWayHashing;
+import org.sunbird.common.request.Request;
+import org.sunbird.helper.ServiceFactory;
+import org.sunbird.learner.util.Util;
+
+public class UserSkillManagementActorTest {
```

Comment:
Unlike functional tests, unit tests should mock dependent systems (e.g. Cassandra, Elastic Search etc.). Rewrite these tests by using power mockito.